config/creatures.cfg

```diff
@@ -13,9 +13,6 @@
 #                     workers will dig and claim tiles. When idle, fighters with a digRate > 0 can dig tiles as indicated
```

Comment:
I have created the tools for windows and uploaded to our repo. I will go with the wiki when I can.
Anyway, I was planning on doing this for a while. We don't have much choice as it will inevitably create problems to have nodes with different scales for every creature. For example, you need to attach an entity to a bone (not a node) and, thus, the same weapon cannot be added to different models. Moreover, I'm sure we will hit other problems with particle effects or whatever when we will add effects to models (as the effects will also be resized).

IMHO, the best is to have everything at the same size.
And, concerning modders, the way I see it, they should use the main menu screen for their tests. The creatures will be bigger, allowing to check the animations are ok, and it will be possible to add them next to another creature to have a reference for size.
I will also explain how to do that in the wiki. But IMHO, it is much more user friendly than to have to manually edit a level to add some new creature or to hack an existing one, to launch a game and pray for the expected one to spawn.